app/models/operator/requests/UpdatePlatformOperatorRequest.scala

```diff
@@ -16,23 +16,96 @@
-                                          subscriptionId: String,
-                                          operatorId: String,
-                                          operatorName: String,
-                                          tinDetails: Seq[TinDetails],
-                                          businessName: Option[String],
-                                          tradingName: Option[String],
-                                          primaryContactDetails: ContactDetails,
-                                          secondaryContactDetails: Option[ContactDetails],
-                                          addressDetails: AddressDetails,
-                                          notification: Option[Notification]
-                                        )
+case class UpdatePlatformOperatorRequest(subscriptionId: String,
+                                         operatorId: String,
+                                         operatorName: String,
+                                         tinDetails: Seq[TinDetails],
+                                         businessName: Option[String],
+                                         tradingName: Option[String],
+                                         primaryContactDetails: ContactDetails,
+                                         secondaryContactDetails: Option[ContactDetails],
+                                         addressDetails: AddressDetails,
+                                         notification: Option[Notification])
 
 object UpdatePlatformOperatorRequest {
-
   implicit lazy val writes: OWrites[UpdatePlatformOperatorRequest] = Json.writes
+
+  def build(answers: UserAnswers, dprsId: String, operatorId: String): EitherNec[Query, UpdatePlatformOperatorRequest] =
+    (
+      answers.getEither(BusinessNamePage),
+      getTradingName(answers),
+      getUkTinDetails(answers),
+      getPrimaryContact(answers),
+      getSecondaryContact(answers),
+      getAddressDetails(answers)
+    ).parMapN { (operatorName, tradingName, tinDetails, primaryContact, secondaryContact, addressDetails) =>
+      UpdatePlatformOperatorRequest(dprsId, operatorId, operatorName, tinDetails, None, tradingName, primaryContact, secondaryContact, addressDetails, None)
+    }
+
+  private[requests] def getTradingName(answers: UserAnswers): EitherNec[Query, Option[String]] =
+    answers.getEither(HasTradingNamePage).flatMap {
+      case true => answers.getEither(TradingNamePage).map(Some(_))
+      case false => Right(None)
+    }
+
+  private[requests] def getUkTinDetails(answers: UserAnswers): EitherNec[Query, Seq[TinDetails]] =
+    answers.getEither(UkTaxIdentifiersPage).flatMap(identifiers => identifiers.toSeq.parTraverse(getUkTin(_, answers)))
+
+  private[requests] def getUkTin(identifier: UkTaxIdentifiers, answers: UserAnswers): EitherNec[Query, TinDetails] = identifier match {
+    case Utr => answers.getEither(UtrPage).map(tin => TinDetails(tin, TinType.Utr, "GB"))
```

Comment:
use a constant for "GB"

Reply:
Ooops. Forgot to do this on this file. It is done now.